.travis.yml

```diff
@@ -41,7 +49,7 @@ before_script:
 script:
   - flake8 --exclude migrations,opentreemap/settings/local_settings.py,*.pyc opentreemap
   - npm run check
-  - testem ci -l Firefox
```

Comment:
The `test` script in `package.json` runs `testem ci` and our travis config was running `testem ci -l Firefox`. Are we intentionally dropping the `-l Firefox`?